README.md

```diff
@@ -34,14 +34,24 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+The responsive website adjusts depending on what screen you are using, like tablet, desktop, phone, etc. The adaptive depends on what screen size is being used and is fixed, you have to write out each design individually.
+
```

Comment:
the major difference is that responsive uses responsive units, vs. adaptive uses hard pixels.  at each media query break point, adaptive snaps suddenly to a new view, and responsive is more fluid as you expand and shrink